ui/src/app/components/gke-app-configuration-panels/create-cromwell.spec.tsx

```diff
@@ -138,7 +140,33 @@ describe(CreateCromwell.name, () => {
     );
   });
 
-  it('should render a DeletePersistentDiskButton when a disk is present but no app', async () => {
+  it('should allow deleting the environment when an app is running', async () => {
```

Comment:
alt: move these tests to a common create-gke-app.spec ?  They're very repetitive